samples/python/console/speech_sample.py

```diff
@@ -843,6 +844,44 @@ def pronunciation_assessment_from_microphone():
                 print("Error details: {}".format(cancellation_details.error_details))
 
 
+def get_reference_words(wave_filename, reference_text, language):
+    audio_config = speechsdk.audio.AudioConfig(filename=wave_filename)
+    speech_config = speechsdk.SpeechConfig(subscription=speech_key, region=service_region)
+    speech_recognizer = speechsdk.SpeechRecognizer(
+        speech_config=speech_config, language=language, audio_config=audio_config
+    )
+
+    # Create pronunciation assessment config, set grading system, granularity and if enable miscue based on your requirement.
+    enable_miscue = True
```

Comment:
In order to get the result with word segmentation, `enable_miscue` should be set as `false`. Otherwise, some reference words will be ignored.

Reply:
The content in short wave files is different from that in reference text, we need to use error_type to filter out some insertion words?

Reply:
Set `enable_miscue` as `False` will use linear grammar and to a force alignment, which means all reference words will be segmented. Here `error_type` is not useful.

Reply:
If set enable_miscue as False, PA engine will return speechsdk.ResultReason.NoMatch, as the content of the short wave file is totally different from what is in the reference text, unless we use the first few seconds wave data from the long wave file which matches the reference text.

Reply:
Got it. Let's have a test with more samples and make sure that setting `enable_miscue` as `True` can get the word segmentation results.